mobile/src/stateMachine/views/qol/qolQuestion.tsx

```diff
@@ -30,66 +30,73 @@ export class QolQuestion extends ViewState {
         return AppViewModel.Instance.QOL;
     }
 
-    async start() {}
-
-    private saveProgress = async () => {
-        await this.viewModel.saveSurveyProgress(this.persona.qolMags);
-        this.cancel();
+    async start() {
+        // Not yet implemented
```

Comment:
I think we can delete this altogether if it's not used.

Reply:
It is required by `ViewState`